connectors/src/connectors/microsoft/index.ts

```diff
@@ -495,3 +509,26 @@ export async function getClient(connectionId: NangoConnectionId) {
     authProvider: (done) => done(null, msAccessToken),
   });
 }
+
+export async function retrieveChildrenNodes(
```

Comment:
yeah I think that should be in the MicrosoftNodeResource
I think there's a fetchChildren already there btw